version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.1"
```

Comment:
Save it as snapshot, sbt-release will change the version to the release version you'll chose then put it back to the next snapshot.
As is, next run of `publish` would publish a release.